### PR TITLE
SQL: method migrate to post request

### PR DESCRIPTION
**Pull request by unrealwork**
### Review tasks

### CODE REVIEW

File: src/test/java/com/axibase/tsd/api/method/sql/SqlMethod.java [hunk 1]
```diff
@@ -43,11 +46,12 @@
...
         logger.debug("SQL query : {}", sqlQuery);
+        Form form = new Form();
+        form.param("q", sqlQuery);
+        form.param("outputFormat", outputFormat.toString());
         return httpSqlApiResource
-                .queryParam("q", sqlQuery)
-                .queryParam("outputFormat", outputFormat.toString())
                 .request()
-                .get();
+                .post(Entity.entity(form, MediaType.APPLICATION_FORM_URLENCODED));
```
Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/sql/SqlMethod.java]: can you use `Entity.form(form)` instead?

Comment (unrealwork) [inline src/test/java/com/axibase/tsd/api/method/sql/SqlMethod.java]: I have replaced it to form

**Comment by Sild** [comment]
ok

**Comment by KutuzovGregory** [comment]
Ok

**Pull request merged**